index.html

```diff
@@ -0,0 +1,231 @@
+		<!-- smoothness library improves the appearance of the datepicker -->
+		<link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+		<script src="https://cdn.mxpnl.com/libs/mixpanel-platform/build/mixpanel-platform.v0.latest.min.js"></script>
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/chosen/1.1.0/chosen.jquery.min.js"></script>
+		<script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.11/js/jquery.dataTables.js"></script>
+		<script src="customQueryGenerator.js"></script>
+		<script src="exportToCSV.js"></script>
+		<script src="generalFunctionsAndVariables.js"></script>
+		<script src="graphFunctions.js"></script>
+	</head>
+	<body class="mixpanel-platform-body">
+		<div class="mixpanel-platform-section">
+			<!-- Event Select -->
+			<div id="labelSelectEvent" class="mixpanel-platform-label" 
+				style="width: 258px; padding: 10 px">Select Event</div>
+			<div id="eventSelect" style="float: left;"></div>
+			
+			<!-- Select Operating System -->
+			<div style="clear: both;"></div>
+			<div id="labelSelectPlatforms" class="mixpanel-platform-label" 
+				style="width: 258px; padding: 10 px">Select Platform(s)</div>
+			<select data-placeholder="Select Platform(s), leave empty for all" 
+				class="chosen-platform" multiple="true" style="width: calc(100% - 300px); float:right" ></select>
+			
+			<!-- Select App Name(s) -->
+			<div style="clear: both;"></div>
+			<div id="labelSelectAppNames" class="mixpanel-platform-label" style=
+				"width: 258px; padding: 10 px">Select Application(s)</div>
+			<select data-placeholder="Select App(s), leave empty for all" class=
+				"chosen-appName" multiple="true" style="width: calc(100% - 300px); float:right" ></select>
+			<!-- Select Device Models -->
+			<div style="clear: both;"></div>
+			<div id="labelSelectDeviceModels" class="mixpanel-platform-label" 
+				style="width: 258px; padding: 10 px">Select Device Model(s)</div>
+			<select data-placeholder="Select Model(s), leave empty for all" class=
+				"chosen-model" multiple="true" style="width: calc(100% - 
+				300px); float:right" ></select>
+			<div style="clear: both;"></div>
+			<!-- Select Number of Devices to be Shown -->
+			<div id="labelNumDevices" class="mixpanel-platform-label" style="width: 
+				258px; padding: 10 px">Show Number of Devices</div>
+			<div id="numResultsSelect" style="float: left;"></div>
+			
+			<div style="padding:10px; float: left; width: 175px;"><input 
+				style="display: none;" type="text" class = "tbxNumSelect" 
+				id="tbxNumResultsSelect" placeholder="Enter # of Device
+				Model(s)"></input></div>
+			<button style= "display: none; float:left; padding: 5px;" class
+				= "tbxNumSelect" id="btnNumDevicesSubmit">Submit</button> 
+			<!-- Select Dates -->
+			<div style="clear: both;"></div>
+			<div id="labelSelectDates" class="mixpanel-platform-label" style=
+				"width: 258px; padding: 10 px">Select Time Period</div>

```

Comment:
Minor point, but if you define some CSS styles, you can reuse them here instead of defining a style= thing every time.

For instance, if you create a new CSS class called "foo", you could just do:

<div id="labelSelectDates" class="mixpanel-platform-label foo" ...

And it'll apply both of them.